ceno_zkvm/src/tables/ops.rs

```diff
@@ -87,7 +87,49 @@ impl OpsTable for PowTable {
     }
 
     fn content() -> Vec<[u64; 3]> {
-        (0..Self::len() as u64).map(|b| [2, b, 1 << b]).collect()
+        (0..Self::len() as u64)
+            .map(|exponent| [2, exponent, 1 << exponent])
+            .collect()
+    }
+
+    fn pack(base: u64, exponent: u64) -> u64 {
+        assert_eq!(base, 2);
+        exponent
+    }
+
+    fn unpack(exponent: u64) -> (u64, u64) {
```

Comment:
does pack & unpack only for test purpose? if yes probably we can define in test mod, or mark function with cfg[test]

Reply:
This is how `LkMultiplicity` finds the table index: https://github.com/scroll-tech/ceno/blob/846033ad805ae115507ed6923acf37e3ff478279/ceno_zkvm/src/witness.rs#L152

Reply:
I c, thanks! now I realized "override" means